scripts/debug

```diff
@@ -0,0 +1,146 @@
+# are met:
+# 1. Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+# 2. Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+# 3. Neither the name of the University nor the names of its contributors
+#    may be used to endorse or promote products derived from this software
+#    without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+# ``AS IS'' AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT 
+# LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR 
+# A PARTICULAR PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE HOLDERS OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, 
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, 
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR 
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF 
+# LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING 
+# NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS 
+# SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+# This fork is Copyright: 2020 Umbrel.
+
+[[ -f "/etc/default/umbrel" ]] && source "/etc/default/umbrel"
+
+echo "====================="
+echo "= Umbrel debug info ="
+echo "====================="
+
+UMBREL_ROOT="$(dirname $(readlink -f "${BASH_SOURCE[0]}"))/.."
+cd "${UMBREL_ROOT}"
+echo
+echo "Umbrel version"
+echo "---------------"
+cat info.json | jq .version | sed 's/"//g'
+
+if [[ ! -z "${UMBREL_OS:-}" ]]; then
+    echo
+    echo "Umbrel OS version"
+    echo "-----------------"
+    echo $UMBREL_OS
+
+    echo
+    echo "Raspberry Pi Model"
+    echo "------------------"
+
+    cat /proc/cpuinfo | tail -3
+
+    echo
+    echo "Firmware"
+    echo "--------"
+
+    vcgencmd version
+
+    echo
+    echo "Tempature"
+    echo "---------"
+
+    vcgencmd measure_temp
+
+    echo
+    echo "Throttling"
+    echo "----------"
+
+    vcgencmd get_throttled
+
+    echo
+    echo "Startup status"
+    echo "--------------"
+    systemctl status umbrel-startup
+    systemctl status umbrel-external-storage
+fi
+echo
+echo "Filesystem information"
+echo "----------------------"
+
+df -h
+echo
+cat /proc/swaps
+
+
+echo
+echo "Networking Information"
+echo "----------------------"
+echo
+
+ifconfig | sed -e "s/\([0-9a-fA-F]\{1,4\}:\)\{7,7\}[0-9a-fA-F]\{1,4\}/y.y.y.y.y.y.y.y/g" | sed -e "s/[0-9a-fA-F]\{1,4\}:\(:[0-9a-fA-F]\{1,4\}\)\{1,4\}/y::y.y.y.y/g" | sed -e "s/\([0-9]\{1,3\}\.\)\{3,3\}[0-9]\{1,3\}/x.x.x.x/g" | sed -e "s/\([0-9a-fA-F]\{2,2\}\:\)\{5,5\}[0-9a-fA-F]\{2,2\}/m.m.m.m/g"
+
+echo
+echo "Docker logs"
+echo "-----------"
+
+docker-compose logs --tail=100
+
+if [[ ! -z "${UMBREL_OS:-}" ]]; then
```

Comment:
It's safe to remove this entire `if` block since `umbrel-external-storage.service` already performs these sanity checks for no drive/more than 1 drive and we can spot any errors in its logs.

Reply:
I added this because it then can give the output of the check and directly tell the user the problem so they don't have to ask.  (Lines 140 - 152)

Reply:
Ah, okay. Yeah that makes sense!